mawabot/utils.py

```diff
@@ -103,6 +103,24 @@ def unload_cog(self, cogname):
             embed = discord.Embed(color=discord.Color.green(), description=f'```{cogname}```')
             embed.set_author(name='Reloaded')
             await ctx.send(embed=embed)
+    
+    @commands.command()
+    async def cogs(self, ctx):
+        '''
+        List the cogs that are currently loaded
+        '''
+
+        msg = '```yaml\nCogs Loaded:\n'
+
+        if self.bot.cogs:
+            for cog in self.bot.cogs:
+                msg += f' - {cog}\n'
+        else:
+            msg += ' - None\n'
+
+        msg += '```'
+
+        await ctx.message.edit(content=msg)
```

Comment:
You should use `'\n'.join(...)` instead of `+=`.

```py
lines = ['```yaml', 'Cogs Loaded:']

if self.bot.cogs:
    for cog in self.bot.cogs:
        lines.append(f' - {cog}')
else:
    lines.append(' - None')

await ctx.message.edit(content='\n'.join(lines))